src/disco/components/Addon/index.js

```diff
@@ -299,13 +309,27 @@ export class AddonBase extends React.Component {
             />
             {this.getDescription()}
           </div>
-          {/* TODO: find the courage to remove {...this.props} */}
-          <InstallButton
-            {...this.props}
-            className="Addon-install-button"
-            defaultInstallSource={defaultInstallSource}
-            size="small"
-          />
+          {_config.get('enableNewInstallButton') ? (
+            <NewInstallButton
+              addon={addon}
+              className="Addon-install-button"
+              defaultInstallSource={defaultInstallSource}
+              disabled={!compatible}
+              enable={enable}
+              install={install}
+              installTheme={installTheme}
+              puffy={false}
+              status={status || UNKNOWN}
+              uninstall={uninstall}
```

Comment:
I think we need to pass in `hasAddonManager` (true) as prop here, right? Right now I don't think it works without this..

Reply:
oh heh, good catch!

Reply:
I am wondering if that should also be part of the check of using the New Button..? Or what happens if it doesn't have that? does it fallback to the trigger method that was used before? or does it just do nothing?

Reply:
fixed!

Reply:
yep, I see 👍